ansible/roles/test/templates/vnet_config.j2

```diff
@@ -25,100 +98,162 @@ vnet_id_list:
 vnet_v6_base: {{ vnet_v6_base }}
 {% endif %}
 
+{# Vnet - Peer Configurations #}
 vnet_peer_list:
-  - Vnet3: Vnet4
-  - Vnet4: Vnet3
-  - Vnet7: Vnet8
-  - Vnet8: Vnet7
+{% for vnet_batch in range (1, 1 + num_vnet)|batch(4) %}
+  - Vnet{{ vnet_batch[2] }}: Vnet{{ vnet_batch[3] }}
+  - Vnet{{ vnet_batch[3] }}: Vnet{{ vnet_batch[2] }}
+{% endfor %}
 
+{# Vnet - Interface Configurations #}
```

Comment:
Can we have this set to max `128`?

Reply:
Can you please review if the changes to limit to 128 are fine? I still have to validate and see if the total routes exceeds the configured "num_routes" with the introduction of new sets of peers.

Reply:
Changes looks ok to me.